rrfs-test/scripts/link_mpasjedi_expr.sh

```diff
@@ -7,7 +7,7 @@ if [[ -z ${RDASApp} ]]; then
 fi
 
 #RDASApp="/path/to/RDASApp"  # set this variable if line2-7 was removed
-BUMPLOC="conus12km-401km11levels"
+BUMPLOC="conus12km-401km11levels_updatesaber"
```

Comment:
As stated above, we change the links under fix/ and don't need to change anything here.